### ISSUE TITLE
Loonas smart curtain

### ISSUE THREAD

**Issue by x5500** [make-all/tuya-local#367]
Logger: custom_components.tuya_local.config_flow
Source: custom_components/tuya_local/config_flow.py:42
Integration: tuya_local ([documentation](https://github.com/make-all/tuya-local), [issues](https://github.com/make-all/tuya-local/issues))
First occurred: 01:01:22 (2 occurrences)
Last logged: 01:01:22

Device matches None with quality of 0%. DPS: {'updated_at': 1675202479.5102003, '1': 'stop', '2': 100, '3': 100, '5': 'back'}
Report this to https://github.com/make-all/tuya-local/issues/

Device Config used:
[smart_curtain.yaml.txt](https://github.com/make-all/tuya-local/files/10551526/smart_curtain.yaml.txt)

TuYa Cloud:
[Smart Curtain.txt](https://github.com/make-all/tuya-local/files/10551512/Smart.Curtain.txt)

**Comment by x5500** [comment]
Hello, Jason!
Thank you for fast reply.
I have tested this update (and also noticed mistakes I've made in my config, sorry for this).
Result the same:

2023-02-02 12:11:27.331 WARNING (MainThread) [custom_components.tuya_local.config_flow] Device matches None with quality of 0%. DPS: {'updated_at': 1675329083.3586822, '1': 'stop', '2': 2, '3': 2, '5': 'back'}
2023-02-02 12:11:27.334 WARNING (MainThread) [custom_components.tuya_local.config_flow] Report this to https://github.com/make-all/tuya-local/issues/

Also i've tried to update secondary entity with back/forward mapping to true/false.. and got same report.
  - entity: switch
    name: Motor reverse mode
    category: config
    icon: "mdi:arrow-u-down-left"
    dps:
      - id: 5
        type: boolean
		name: control_back_mode
        mapping:
          - dps_val: forward
            value: false
          - dps_val: back
            value: true
------
For future use i have extracted from Tyua cloud correct product ID and name: 
id: wraaecjqfiirj8go 
name: Smart curtain
